### PR TITLE
feat: add prop to specify ens chain

### PR DESCRIPTION
**Pull request by danielr18**
Allow changing which chain is used for ENS, by setting the `ensChain` prop on `RainbowKitProvider`

Relates to: https://github.com/rainbow-me/rainbowkit/discussions/752

### CODE REVIEW

File: .changeset/rude-fireants-doubt.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'example': patch
```
Comment (markdalgleish) [inline .changeset/rude-fireants-doubt.md:2]: This should be removed.

File: packages/example/pages/_app.tsx [hunk 2]
```diff
@@ -380,6 +382,30 @@ function RainbowKitApp({ Component, pageProps }: AppProps) {
                         </select>
                       </td>
                     </tr>
+                    <tr>
+                      <td>ensChain</td>
```
Comment (markdalgleish) [inline packages/example/pages/_app.tsx:386]: Thanks for adding this!

**Comment by markdalgleish** [comment]
Thanks for the PR!

This will need some docs on the "Chains" page: https://www.rainbowkit.com/docs/chains — did you want to write these, or would you prefer push updates to your PR?

File: .changeset/rude-fireants-doubt.md (hunk 1 above)
Comment (danielr18) [reply .changeset/rude-fireants-doubt.md]: Removed

**Comment by danielr18** [comment]
Thanks, I've added a section in the docs.

**Review by DanielSinclair — COMMENTED** [review]
A bit unsure of the API design here. It may be a better UX for RainbowKit to resolve ENS for the user's selected chain or `initialChain`, as opposed to bubbling up an ENS-specific prop to the dApp dev. Can you walk me through the design decisions?

**Comment by DanielSinclair** [comment]
Will close this for now because there might be a better/simpler means to an end for this particular testnet ENS problem. Can reopen if this changes or based on additional conversations here.

**Pull request closed without merging**